plugins/github/deploy.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Smart

Reply:
Except that it doesn't work, because it only downloads .mk files when you update brokkr in the project that uses this :/

Reply:
Will look into that

Reply:
Hmm... That is true. The Airflow one downloads within the `airflow.mk` https://github.com/unacast/brokkr/blob/master/plugins/airflow/airflow.mk#L94

But it isn't quite nice that either. Maybe if we downloaded the full "folder" instead. Although local paths, AND figuring out every file that is within the folder using curl agains https://github.com is perhaps a challenge.